ast/builtins.go

```diff
@@ -766,6 +783,20 @@ var Replace = &Builtin{
 	),
 }
 
+// ReplacerReplace replaces a string from a list of old, new string pairs.
+// Replacements are performed in the order they appear in the target string, without overlapping matches.
+// The old string comparisons are done in argument order.
+var ReplacerReplace = &Builtin{
+	Name: "replacer.replace",
```

Comment:
We don't have to match the Go package names 1:1. I'd prefer something like `strings.replace_n` here.

Reply:
I agree. I'll go with `strings.replace_n`